### DISCUSSION TITLE
Error ip addresses

### DISCUSSION THREAD

**Discussion by mvb-helpdesk** [netbox-community/netbox#8221] [Q&A] [answered]
After Upgrade from 3.04. to 3.1.4 I get the following error message when I want to display the associated ip addresses for a network.
IPAM > Prefixes > Network> IP Addresses.

<class 'ValueError'>

'role' does not resolve to an item that supports prefetching - this is an invalid parameter to prefetch_related().

Python version: 3.9.6
NetBox version: 3.1.4

**Comment by candlerb** [comment, marked as answer]
Bug, already reported: #8213
